src/stylesheets/components/_buttons.scss

```diff
@@ -111,29 +93,31 @@
   border-radius: 0 4px 4px 0;
 }
 
-/*--- Active styles ---*/
-
-.open > .btn--ghost--inverse {
-  @include button($white, $grey-1, $grey-2);
-}
-
-/*--- Toggle styles ---*/
+/*--- Toggles ---*/
 
 .toggle--menu {
-  > [class*="btn--"] {
-    margin-bottom: 0;
-  }
   .btn--close {
+    display: none;
     float: right;
     @include links($grey-1, none, $grey-2, $grey-1);
     @include milli;
     margin-bottom: 0;
   }
+
+  &.open > .btn--ghost--inverse {
+    @include button($white, $grey-1, $grey-2);
+  }
 }
 
-body:not(.has-js) .toggle--menu {
-  > [class*="btn--"],
-  .btn--close {
-    display: none;
+.has-js .toggle--menu .btn--close {
+  display: inline-block;
+}
+
+a.toggle[class*="btn--"],
+ .toggle[class*="btn--"] {
```

Comment:
Amend to `btn--toggle`. Details mentioned in Jira task https://ukparliament.atlassian.net/browse/WEBDEV-695?oldIssueView=true